src/cloud-api-adaptor/install/overlays/aws/kustomization.yaml

```diff
@@ -63,7 +63,6 @@ secretGenerator:
 ##TLS_SETTINGS
 
 patchesStrategicMerge:
-  #- cri_runtime_endpoint.yaml # set (modify host's runtime cri socket path in the file, default is /run/containerd/containerd.sock)
```

Comment:
Hi @stevenhorsman ! This entire commit can be squashed on the previous (that removes the cri runtime endpoint) so in case of revert, we revert it atomically. Or do you prefer it separated?

Reply:
Yeah - as you can see from the discussions I had above with Beraldo I really struggled in this one to balance between having separate commits for different areas, like we guide in CoCo and mandate in kata-containers whilst making atomic commits (I already reduced from 50 commits to 25 as we have now). Realistically I don't see a scenario where we are likely to back out to the middle of this commit sequence, but I'm not against bending our rules to make something like a bisect easier to do. It probably means we should push back on lots of kata PRs for the same reasons though!

Reply:
Done